chatbot/settings.py

```
@@ -0,0 +1,162 @@
+        'ENABLE_DRUG_DATABASE': False,
+        'ENABLE_APPOINTMENT_SYSTEM': False,
+        'API_TIMEOUT': 10,  # ثانیه
+    }
+}
+
+# ادغام تنظیمات از فایل settings.py اصلی
+if hasattr(settings, 'CHATBOT_SETTINGS'):
+    CHATBOT_SETTINGS.update(settings.CHATBOT_SETTINGS)
+
+# تنظیمات محیط (Environment Variables)
+def get_env_setting(key: str, default=None, cast_type=str):
```

Ruff (0.12.2)

Missing return type annotation for private function `get_env_setting`

(ANN202)